core/commands/object/patch.go

```diff
@@ -145,14 +145,14 @@ Example:
 
 var patchRmLinkCmd = &oldcmds.Command{
 	Helptext: cmdkit.HelpText{
-		Tagline: "Remove a link from an object.",
+		Tagline: "Remove a link from a given object.",
 		ShortDescription: `
-Removes a link by the given name from root.
+Remove a Merkle-link from the given object and return the hash of the result.
 `,
 	},
 	Arguments: []cmdkit.Argument{
 		cmdkit.StringArg("root", true, false, "The hash of the node to modify."),
-		cmdkit.StringArg("link", true, false, "Name of the link to remove."),
+		cmdkit.StringArg("name", true, false, "Name of the link to remove."),
```

Comment:
While I prefer this new flat, it breaks the API and I'm not convinced it's really worth the potential issues.

Reply:
@Stebalien oh interesting, I assumed the name of positional args wasn't part of a public API. Out of curiosity, where does cause problems? Is there a test for it?